scripts/deploy_oak_functions_api

```diff
@@ -53,6 +53,11 @@ sed \
 readonly CONFIG_ID='weather-lookup-config'
 readonly API_ID='weather-lookup-api'
 
+# First delete the existing api-config and gateway
```

Comment:
It is perhaps a good idea to add this code in an if statemet that runs it is the API already exists, then there is no need to keep it commented out.

Reply:
Thanks. Fixed.